CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,135 @@
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is representing the community in public spaces.
+
+Our community consists of contributors in their efforts related to the development and distribution of any materials for the HoloViz organization. The projects in this community include the repositories in the [HoloViz GitHub organization](https://github.com/holoviz), such as Param, HoloViews, hvPlot, Panel, GeoViews, Lumen, Datashader, and Colorcet. Other efforts, whether online or in-person, undertaken by this community and coordinated through [HoloViz communication channels](https://holoviz.org/community.html) are also covered under this Code of Conduct. This code applies equally to founders, developers, mentors and new community members, in all spaces managed by HoloViz. In addition, violations of this code outside these spaces may affect a person’s ability to participate within them.
+
+## Reporting
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the HoloViz Code of Conduct committee at
+[coc@holoviz.org](mailto:coc@holoviz.org), or by filling out [TODO: this form]() which can be submitted anonymously. Currently, the committee consists of:
+* James Bednar [@jbednar](https://github.com/jbednar)
+* Marc Skov Madsen [@MarcSkovMadsen](https://github.com/MarcSkovMadsen)
+* Philipp Rudiger [@philippjfr](https://github.com/philippjfr)
+
+If your complaint involves anyone who is on this list, please reach out separately to one or more members of the list who have not been involved in the behavior in question. If a report involves a member of the listed committee, that member will not participate in the investigation or any decisions related to that report.
+
+If you choose to email instead of using the form, please include the following information:
+
+* Your contact info (if you would like us to be able to follow up or collect additional information)
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, try to include them as well
+* When and where the incident occurred. Please be as specific as possible.
```

Comment:
This is the only bullet point that ends in a full stop, which is incongruous.

Reply:
thanks! done